### PR TITLE
Overlay Updates

### PR DESCRIPTION
**Pull request by Iamlooker**

### CODE REVIEW

**Comment by phhusson** [comment]
`Fatal: overlay.mk: Keep entries sorted`

**Comment by Iamlooker** [comment]
Last one is for Asus Zenfone Max M2

File: Asus/ZenFoneMaxM2/res/values-land/notch.xml [hunk 1]
```diff
@@ -1,22 +0,0 @@
...
-** Unless required by applicable law or agreed to in writing, software
-** distributed under the License is distributed on an "AS IS" BASIS,
-** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-** See the License for the specific language governing permissions and
-** limitations under the License.
-*/
--->
-<resources>
-    <!-- Height of the status bar -->
-    <dimen name="status_bar_height">24dp</dimen>
```
Comment (phhusson) [inline Asus/ZenFoneMaxM2/res/values-land/notch.xml:21]: why?

File: Asus/ZenFoneMaxM2/res/values/config.xml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        <item>125</item>
+        <item>136</item>
+        <item>146</item>
+        <item>188</item>
+        <item>217</item>
+        <item>241</item>
+        <item>251</item>
+        <item>256</item>
+        <item>260</item>
+        <item>264</item>
```
Comment (phhusson) [inline Asus/ZenFoneMaxM2/res/values/config.xml:25]: sounds wrong. Auto brightnesss works on Q with this?

File: Asus/ZenFoneMaxM2/res/values-land/notch.xml (hunk 1 above)
Comment (Iamlooker) [reply Asus/ZenFoneMaxM2/res/values-land/notch.xml:21]: it is done twice once in values_land and once in values

File: Asus/ZenFoneMaxM2/res/values/config.xml [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    </integer-array>
+    <bool name="config_bluetooth_le_peripheral_mode_supported">true</bool>
+    <bool name="config_showNavigationBar">true</bool>
+    <bool name="config_powerDecoupleAutoSuspendModeFromDisplay">true</bool>
+    <bool name="config_useDevInputEventForAudioJack">true</bool>
+    <bool name="config_supportAudioSourceUnprocessed">true</bool>
+    <bool name="config_sustainedPerformanceModeSupported">false</bool>
+    <bool name="config_wifi_batched_scan_supported">false</bool>
+    <bool name="config_lidControlsSleep">false</bool>
+    <bool name="config_device_volte_available">true</bool>
```
Comment (phhusson) [inline Asus/ZenFoneMaxM2/res/values/config.xml:66]: useless but well

File: Asus/ZenFoneMaxM2/res/values/config.xml [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    <bool name="config_device_vt_available">true</bool>
+    <bool name="config_automatic_brightness_available">true</bool>
+    <bool name="config_allowAutoBrightnessWhileDozing">true</bool>
+    <bool name="config_displayBlanksAfterDoze">true</bool>
+    <bool name="config_dozeAfterScreenOff">true</bool>
+    <bool name="config_dozeAfterScreenOffByDefault">false</bool>
+    <bool name="config_dozeAlwaysOnDisplayAvailable">true</bool>
+    <bool name="config_enableBurnInProtection">true</bool>
+    <integer name="config_shutdownBatteryTemperature">650</integer>
+    <integer name="config_screenBrightnessSettingMinimum">3</integer>
```
Comment (phhusson) [inline Asus/ZenFoneMaxM2/res/values/config.xml:76]: Sounds high, you're sure brightness 1 isn't usable?

File: Asus/ZenFoneMaxM2/res/values/config.xml [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+    <integer name="config_screenBrightnessSettingDefault">62</integer>
+    <integer name="config_screenBrightnessSettingMaximum">255</integer>
+    <integer name="config_screenBrightnessDoze">2</integer>
+    <integer name="config_screenBrightnessDark">2</integer>
+    <integer name="config_screenBrightnessDim">2</integer>
```
Comment (phhusson) [inline Asus/ZenFoneMaxM2/res/values/config.xml:81]: Weird to have dim lower than minimum

File: Asus/ZenFoneMaxM2/res/xml/power_profile.xml [hunk 6]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<device name="Android">
+    <item name="screen.on">63</item>
+    <item name="screen.full">261</item>
+    <array name="cpu.speeds">
```
Comment (phhusson) [inline Asus/ZenFoneMaxM2/res/xml/power_profile.xml:5]: No modem? bluetooth? Wifi?

File: Elephone/Solider/AndroidManifest.xml [hunk 7]
```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.Elephone.Solider"
+        android:versionCode="29"
+        android:versionName="10">
+        <overlay android:targetPackage="android"
+                android:requiredSystemPropertyName="ro.vendor.build.fingerprint"
+		android:requiredSystemPropertyValue="+Elephone/Soldier/Soldier:8.0.0/O00623/20180823:user/release-keys*"
```
Comment (phhusson) [inline Elephone/Solider/AndroidManifest.xml:7]: Is it really useful to match to this precise vendor version?
Sounds counter-productive

Comment (Iamlooker) [reply Elephone/Solider/AndroidManifest.xml:7]: i used shortened name and send him the apk but he said it was not working

Comment (phhusson) [reply Elephone/Solider/AndroidManifest.xml:7]: Well that's wrong

Comment (Iamlooker) [reply Elephone/Solider/AndroidManifest.xml:7]: so should i use "+Solider" only?

File: Asus/ZenFoneMaxM2/res/values/config.xml (hunk 4 above)
Comment (Iamlooker) [reply Asus/ZenFoneMaxM2/res/values/config.xml:76]: Ya

File: Asus/ZenFoneMaxM2/res/xml/power_profile.xml (hunk 6 above)
Comment (Iamlooker) [reply Asus/ZenFoneMaxM2/res/xml/power_profile.xml:5]: This was durectly extracted from framework-res and other framework files

File: Asus/ZenFoneMaxM2/res/values/config.xml (hunk 2 above)
Comment (Iamlooker) [reply Asus/ZenFoneMaxM2/res/values/config.xml:25]: Again m not pro and everything was extracted from stock so sorry
And I will try to be better next time ty

File: Asus/ZenFoneMaxM2/res/values/config.xml (hunk 3 above)
Comment (Iamlooker) [reply Asus/ZenFoneMaxM2/res/values/config.xml:66]: Okay I will remove

Comment (Iamlooker) [reply Asus/ZenFoneMaxM2/res/values/config.xml:66]: Okay I will remove

File: Asus/ZenFoneMaxM2/res/values/config.xml (hunk 5 above)
Comment (Iamlooker) [reply Asus/ZenFoneMaxM2/res/values/config.xml:77]: fixed it sir

**Pull request closed without merging**